src/ui/Errors/Loading/Loading.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
I don't really think we need to nest this under `Loading/Loading`. See how forms is done? Let's make this `src/ui/Errors/Loading.js`